aidial_assistant/application/assistant_application.py

```diff
@@ -99,23 +99,24 @@ async def chat_completion(
             buffer_size=self.args.chat_conf.buffer_size,
         )
 
-        addons: list[str] = (
-            [addon.url for addon in request.addons] if request.addons else []  # type: ignore
+        token_source = AddonTokenSource(
+            request.headers,
+            [addon.url for addon in request.addons] if request.addons else [],  # type: ignore
         )
-        token_source = AddonTokenSource(request.headers, addons)
 
         tools: dict[str, PluginInfo] = {}
```

Comment:
`tools`, `addons`, `plugins` within tree lines of code all referencing the same kind of entity. 
We should really stick to one term for consistency and clarity.